src/spider/worker/WorkerClient.cpp

```
@@ -0,0 +1,99 @@
+
+        // Receive response
+        std::optional<msgpack::sbuffer> const optional_response_buffer
+                = core::receive_message(socket);
+        if (!optional_response_buffer.has_value()) {
+            return std::nullopt;
+        }
+        msgpack::sbuffer const& response_buffer = optional_response_buffer.value();
+
+        scheduler::ScheduleTaskResponse response;
+        msgpack::object_handle const response_handle
+                = msgpack::unpack(response_buffer.data(), response_buffer.size());
```

cppcheck (2.10-2)

[performance] Variable 'id' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)